src/indicator/algorithm/exponentialMovingAverage.js

```diff
@@ -1,14 +1,13 @@
-import calculator from './calculator/slidingWindow';
 import d3 from 'd3';
-import merge from './merge';
+import indicator_algorithm_calculator_exponentialMovingAverage from './calculator/exponentialMovingAverage';
+import indicator_algorithm_merge from './merge';
 
 export default function() {
 
-    var ema = calculator()
-            .accumulator(d3.mean)
+    var ema = indicator_algorithm_calculator_exponentialMovingAverage()
```

Comment:
This does feel a little verbose, the value of `indicator_algorithm_calculator_exponentialMovingAverage` versus simply `exponentialMovingAverage` is that it would allow you to locate the source of the module a little more quickly, but it's not much effort to just locate the module via the import statement.